### PR TITLE
Add openSUSE installation instructions

### PR DESCRIPTION
**Pull request by mimi1vx**

### CODE REVIEW

**Comment by emilypi** [comment]
@mimi1vx is this information still good? If so, I can approve and merge

**Comment by mimi1vx** [comment]
@emilypi it only need's update Leap version

**Comment by mimi1vx** [comment]
@emilypi PR updated

File: downloads/linux.markdown [hunk 1]
```diff
@@ -69,6 +69,32 @@ The official repos on Arch Linux contain packages `ghc`, `cabal-install`, `happy
 
 Steps to setup stack are [on the stack website](https://github.com/commercialhaskell/stack/blob/master/doc/install_and_upgrade.md#arch-linux).
 
+### openSUSE Leap
+
+*   Leap 15.1 has ghc-8.6.4 nad cabal-install-2.4.0.0
```
Comment (emilypi) [inline downloads/linux.markdown:74]:
```suggestion
*   Leap 15.1 has ghc-8.6.4 and cabal-install-2.4.0.0
```

Comment (mimi1vx) [reply downloads/linux.markdown:74]: thx

**Comment by TikhonJelvis** [comment]
Thanks for updating this, by the way!

**Pull request merged**